lib/tasks/install.rake

```diff
@@ -26,7 +26,7 @@ module Packaging
     DESTDIR = ENV["DESTDIR"] || "/"
     YAST_DIR = DESTDIR + "/usr/share/YaST2/"
     YAST_LIB_DIR = DESTDIR + "/usr/lib/YaST2/"
-    YAST_ICON_BASE_DIR = DESTDIR + "/usr/share/YaST2/theme/current/icons"
+    YAST_ICON_BASE_DIR = DESTDIR + "/usr/share/icons/"
```

Comment:
Please add a comment that this is only for the module specific icons, the generic icons should still got to `/usr/share/YaST2/theme`.

To avoid confusion with the branding path.

Reply:
Ah, the branding icons already go to `/usr/share/icons`... so OK.